agent/command/expansion_update.go

```diff
@@ -56,12 +61,19 @@ func (c *update) ParseParams(params map[string]interface{}) error {
 		if item.Key == "" {
 			return errors.Errorf("expansion key at index %d must not be a blank string", i)
 		}
+		if item.Value != "" && item.Concat != "" {
```

Comment:
We didn't have this validation before- but we either used concat or value, not both. It also wouldn't make sense to use both. I think forcing this makes user's tests more predictable.